packages/react/src/formElements/MdInput.tsx

```diff
@@ -30,8 +30,8 @@ export interface MdInputProps {
   onKeyDown?(_e: React.KeyboardEvent<HTMLInputElement>): void;
   minLength?: number;
   maxLength?: number;
-  min?: number;
-  max?: number;
+  min?: number | string;
+  max?: number | string;
   step?: number;
 }
```

Comment:
Jeg får typescript-errors med denne endringen! 

ype 'string | number | undefined' is not assignable to type 'number | undefined'.
  Type 'string' is not assignable to type 'number'.ts(2322)

Reply:
Glem det, brukte feil properties!